File: C++/M01_ProgrammingBasics/L02_ConditionalStatements/Exercises/Solutions/P06_GodzillaVsKong.cpp
#include <iostream>
#include <string>
#include <algorithm>
using namespace std;

int main() {

	double budget, clothingPricePerWalkOn;
	int walkOnQuantity;

	cin >> budget >> walkOnQuantity >> clothingPricePerWalkOn;

	double decor = budget * 0.1;

	if (150 < walkOnQuantity) {
		clothingPricePerWalkOn *= 0.9;
	}

	cout.setf(ios::fixed);
	cout.precision(2);

	double clothingExpenses = walkOnQuantity * clothingPricePerWalkOn;
	double movieExpenses = decor + clothingExpenses;

	double amount = abs(budget - movieExpenses);
	if (movieExpenses <= budget) {
		cout << "Action!" << endl;
		cout << "Wingard starts filming with " << amount << " leva left." << endl;
	} else {
		cout << "Not enough money!" << endl;
		cout << "Wingard needs " << amount << " leva more." << endl;
	}

	return 0;
}

File: Python/M01_ProgrammingBasics/L02_ConditionalStatements/Exercises/Solutions/P06_GodzillaVsKong.py
budget = float(input())
staff_number = int(input())
price_per_staff = float(input())

decor = budget * 0.1
price_all_staff = staff_number * price_per_staff

if staff_number > 150:
    price_all_staff = price_all_staff * 0.9

cost = price_all_staff + decor

if budget >= cost:
    remaining = budget - cost
    print('Action!')
    print(f'Wingard starts filming with {remaining:.2f} leva left.')
else:
    remaining = cost - budget
    print('Not enough money!')
    print(f'Wingard needs {remaining:.2f} leva more.')